samples/testsuite-xlt/config/default.properties

```diff
@@ -109,12 +109,16 @@ com.xceptance.xlt.http.responseId.headerName = X-XLT-ResponseId
 ##                              whole lifetime of a WebClient
 ## - shuffleAddresses ......... whether to shuffle multiple resolved addresses
 ## - pickOneAddressRandomly ... whether to use only one out of multiple addresses
+## - ignoreIPv4Addresses ...... whether to ignore IPv4 addresses received from DNS
+## - ignoreIPv6Addresses ...... whether to ignore IPv6 addresses received from DNS
```

Comment:
IMO there is no need/benefit in saying that these addresses where received from DNS since the entire config section is about DNS.

Reply:
I hope being a bit more verbose here helps understanding better what exactly is meant.

Reply:
Yep.